spec/factories/stopwatches.rb

```diff
@@ -0,0 +1,12 @@
+FactoryBot.define do
+  factory :stopwatch do
+    association :user
+    label { Faker::Verb.base } 
+
+    # after(:build) do |stopwatch|
+    #   5.times do
+    #     stopwatch.laps << FactoryBot.build(:lap, stopwatch: stopwatch)
+    #   end
+    # end
```

Comment:
Remove commented code if you don't need it

Reply:
Thanks, I was thinking to do that to test the use/relation with laps but I didn't need it.